README.md

```diff
@@ -173,4 +128,96 @@ now you can see the response body incl. api error message:
     "message" : "[VDC-yy-xxxx] Operation cannot be executed since this Kubernetes Nodepool is already marked for deletion. Current state of the resource is FAILED_DESTROYING."
   }]
 }
-```
\ No newline at end of file
+```
+
+## Building The Provider
```

Comment:
An idea: we could move this information about building, developing, testing in a different file that is relevant only for the contributors of the provider

Reply:
good idea. will leave it only in the Readme in github